src/fin.ts

```diff
@@ -0,0 +1,946 @@
+        {
+          name: "reset",
+          description:
+            "Recreate all or specified project services, their containers and volumes",
+          priority: 70,
+          args: {
+            name: "service",
+          },
+        },
+        {
+          name: ["remove", "rm"],
+          description:
+            "Remove all project services, networks and all their volumes, or specified services only",
+          options: [
+            {
+              name: ["--force", "-f"],
+              description:
+                "Do not ask for confirmation when deleting all project services",
+            },
+          ],
+          args: {
+            name: "service",
+          },
+        },
+        {
+          name: "list",
+          description: "List running Docksal projects (alias: fin pl)",
+          options: [
+            {
+              name: ["--all", "-a"],
+              description: "List all Docksal projects (stopped as well)",
+            },
+          ],
+        },
+        {
+          name: "create",
+          description:
+            "Create a new project with a pre-configured boilerplate: Drupal, Wordpress, Magento, Laravel, Backdrop, Hugo, Gatsby, and others",
+          options: [
+            {
+              name: "--name",
+              description: "Provide project name upfront",
+              requiresEquals: true,
+              args: {
+                name: "name",
+              },
+            },
+            {
+              name: "--choice",
+              description: "Provide software choice number upfront",
+              requiresEquals: true,
+              args: {
+                name: "#",
+              },
+            },
+            {
+              name: "--repo",
+              description:
+                "Clone from a custom repo: name (--choice is set to '0' automatically)",
+              requiresEquals: true,
+              args: {
+                name: "name",
+              },
+            },
+            {
+              name: "--branch",
+              description: "Clone from a custom repo: branch name (optional)",
+              requiresEquals: true,
+              args: {
+                name: "name",
+              },
+            },
+            {
+              name: ["--yes", "-y"],
+              description: "Avoid confirmation",
+            },
+          ],
+        },
+        {
+          name: "config",
+          description: "Show project configuration",
+        },
+        {
+          name: "build",
+          description:
+            "Build or rebuild services (alias for 'docker-compose build')",
+        },
+      ],
+    },
+    {
+      name: "ssh-key",
+      description: "Manage SSH keys (fin help ssh-key)",
+      subcommands: [
+        {
+          name: "add",
+          description:
+            "Add a private SSH key from $HOME/.ssh by file name. mAdds all default keys (id_rsa/id_dsa/id_ecdsa/id_ed25519) if no file name is given",
```

Comment:
Extra `m` in the text 🙂
```suggestion
            "Add a private SSH key from $HOME/.ssh by file name. Adds all default keys (id_rsa/id_dsa/id_ecdsa/id_ed25519) if no file name is given",
```